src/api/index.js

```diff
@@ -49,6 +52,30 @@ const getProfileData = authState => {
   }
 };
 
+const getProfileIdData = (authState, oktaId) => {
+  try {
+    return apiAuthGetId(getAuthHeader(authState), oktaId).then(
+      response => response.data
+    );
+  } catch (error) {
+    return new Promise(() => {
+      console.log(error);
+      return [];
+    });
+  }
+};
+
+const putProfileData = (url, editedData, authState) => {
```

Comment:
You can rename this function to putData. So we could reuse putData for editing other resources.